config/core/test-configs.yaml

```diff
@@ -158,16 +158,41 @@ test_plans:
             - 'lab-broonie'
             - 'lab-collabora-staging'
 
-  chromeos-flash:
+  chromeos-flash-grunt:
     pattern: 'chromeos/chromeos-flash-template.jinja2'
     params:
       job_timeout: '60'
       docker_image: 'kernelci/chromeos-flash'
-      chromeos_image: 'chromiumos_test_image-R94-14150.19.0-octopus.bin'
+      chromeos_image: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/crostest007/chromiumos-grunt/amd64/chromiumos_test_image.bin.gz'
       flashing_kernel: http://storage.chromeos.kernelci.org/images/kernel/v5.17.2-x86_64+x86_chromebook/kernel/bzImage
       flashing_modules: http://storage.chromeos.kernelci.org/images/kernel/v5.17.2-x86_64+x86_chromebook/modules.tar.xz
       flashing_rootfs: http://storage.chromeos.kernelci.org/images/rootfs/chromeos/20220412.0/bullseye-chromeos-flash/amd64/full.rootfs.tar.xz
       flashing_ramdisk: http://storage.chromeos.kernelci.org/images/rootfs/chromeos/20220412.0/bullseye-chromeos-flash/amd64/initrd.cpio.gz
+      flashing_device: detect
+
+  chromeos-flash-hatch:
+    pattern: 'chromeos/chromeos-flash-template.jinja2'
+    params:
+      job_timeout: '60'
+      docker_image: 'kernelci/chromeos-flash'
+      chromeos_image: 'https://storage.chromeos.kernelci.org/images/rootfs/chromeos/crostest007/chromiumos-hatch/amd64/chromiumos_test_image.bin.gz'
```

Comment:
How about adding these parameters to the device type definitions instead? That way we would keep a single `chromeos-flash` test plan and the parameters would be directly tied to each Chromebook.

Reply:
I will check how to do that

Reply:
With the `params` attribute, see this one for example:

```yaml
  meson-g12a-sei510:
    mach: amlogic
    class: arm64-dtb
    boot_method: uboot
    flags: ['fastboot']
    params:
      kernel_addr: '0x01080000'
      dtb_addr: '0x1000000'
      ramdisk_addr: '0x6000000'
      cmdline: '"console=ttyAML0,115200 consoleblank=0 earlycon"'
```

Reply:
Thank you! That will help a lot.

Reply:
This part probably improved as suggested